### PR TITLE
Add note to use --grpc-address flag if the node isn't run on default address

### PR DESCRIPTION
**Pull request by Vuksan**
More info can be found in this [PR](https://github.com/0xPolygon/polygon-sdk/pull/50)

### CODE REVIEW

File: docs/how-tos/howto-query-operator.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ A functioning node is required in order to query any kind of operator informatio
 With the Polygon SDK, node operators are in control and informed about what the node they're operating is doing.<br />
 At any time, they can use the node information layer, built on top of gRPC, and get meaningful information - no log sifting required.
 
+**Note:** If your node isn't running on `127.0.0.1:8545` you should add a flag `--grpc-address <address:port>` to the commands listed in this document.
```
Comment (zivkovicmilos) [inline docs/how-tos/howto-query-operator.md:15]: You can add the note like this, to make it stand out more:
https://github.com/0xPolygon/polygon-sdk-docs/blob/main/docs/overview.md

With :::note

**Review by zivkovicmilos — APPROVED** [review]
Looks good to me 💯

File: docs/how-tos/howto-query-operator.md (hunk 1 above)
Comment (Vuksan) [reply docs/how-tos/howto-query-operator.md:15]: But it isn't parsed by the markdown, I just see three colons and `note` 🤷 
Screen Shot 2021-06-04 at 14 32 33 PM

Comment (zivkovicmilos) [reply docs/how-tos/howto-query-operator.md:15]: We use this trick from docusaurus:
https://docusaurus.io/docs/markdown-features/admonitions

Comment (Vuksan) [reply docs/how-tos/howto-query-operator.md:15]: I see, will do that. Thanks! 🍻

**Pull request merged**